content/guides/workshop/11_what_next.md

```diff
@@ -3,6 +3,8 @@ title: What next after the Docker workshop
 keywords: get started, setup, orientation, quickstart, intro, concepts, containers,
   docker desktop
 description: Making sure you have more ideas of what you could do next with your application
+aliases:
+- /get-started/11_what_next/
```

Comment:
[nit] Not important, but does it matter to users that we skip from 9 to 11 😅

Reply:
Good catch. Updated in a commit below.